src/Koffee/MainModel.fs

```diff
@@ -282,6 +282,8 @@ type CancelToken() =
     member this.IsCancelled = cancelled
     member this.Cancel () = cancelled <- true
 
+type KeyComboCount = int option
```

Comment:
This is one of the things I learnt from some Pluralsight course, that it's _sometimes_ better to alias or create new types even for mundane items. But giving some nights sleep on it, I agree with that this use case is unfitting.

Reply:
Removed it. Now just using `int option` instead.

Would maybe be good to have it as a separate type that enforces that the value is positive, but then it would be tedious to do arithmetic on it.